client/elements/menus/language-base-menu.html

```diff
@@ -7,32 +7,33 @@
 <link rel="import" href="../../bower_components/neon-animation/web-animations.html">
 <link rel="import" href="../../bower_components/app-storage/app-localstorage/app-localstorage-document.html">
```

Comment:
Those should be absolute paths. right  ?

Reply:
correct!